frameworks/Swift/vapor/Sources/vapor-tfb/main.swift

```diff
@@ -0,0 +1,24 @@
+import Vapor
+
+// Services
+var services = Services.default()
+
+// Router
+let router = EngineRouter.default()
+// Routes
+let json = Message(message: "Hello, world!")
+router.get("json") { req in
+    return json.encode(status: .ok, headers: ["Server": "Vapor"], for: req)
+}
+let plaintextRes = HTTPResponse(status: .ok, headers: ["Server": "Vapor", "Content-Type": "text/plain"], body: "Hello, world!")
+router.get("plaintext") { req in
+    return plaintextRes
+}
+services.register(router, as: Router.self)
+
+// Middlewares (remove default mws)
+var middlewares = MiddlewareConfig()
```

Comment:
What middleware is this removing and why?  Is this to improve performance?

Reply:
Yes, by default there is an error reporting middleware enabled. This is not required to create valid HTTP responses and should be removed while benchmarking.

Reply:
Gotcha.  Sometimes there is the temptation for implementations to remove lots of essential middleware that no one would ever disable in a non-benchmark scenario, and we end up benchmarking something meaningless that isn't really the framework that anyone uses.  But it sounds like that is not what's happening here.

Thanks for answering my questions.